aldryn_forms/admin/base.py

```diff
@@ -46,24 +52,30 @@ def get_recipients(self, obj):
 
     def get_recipients_for_display(self, obj):
         people_list = self.get_recipients(obj)
-        html = render_to_string(
-            template_name='admin/aldryn_forms/display/recipients.html',
-            dictionary={'people': people_list}
-        )
+        try:
+            html = render_to_string(
+                template_name='admin/aldryn_forms/display/recipients.html',
+                dictionary={'people': people_list}
+            )
+        except TypeError:
+            html = render_to_string(
+                template_name='admin/aldryn_forms/display/recipients.html',
+                context={'people': people_list}
+            )
```

Comment:
Would be good to have a comment here saying that this is done because of Django 1.10 compat.